extensions/3DTILES_implicit_tiling/README.md

```diff
@@ -0,0 +1,963 @@
+
+```javascript
+function createJumpBuffer(sBuffer) {   
+    let jBuffer = [];
+    let sBufferIndex = 1;
+    let currentLevel = 0;
+    while (sBufferIndex != sBuffer.length) {
+        currentLevel++;
+        jBuffer.push([]);
+        // Get maximum number of elements in this level, based on number of tiles in the level above that will subdivide.
+        let currentLevelMax = jBuffer[currentLevel - 1].length * tilingScheme;
+        jumpBufferSize += currentLevelMax;
+        for (let i = 0; i < currentLevelMax; i++) {
+            if (sBuffer[sBufferIndex++] === 1) {
+                jBuffer[currentLevel].push(jBuffer[currentLevel - 1][Math.floor(i / tilingScheme)] * tilingScheme + (i % tilingScheme));
+            }
+        }
+    }
+    return jBuffer;
+}
+```
+
+#### Tile Index Lookup
+
+For all tile lookups in complete levels, the tile index in the content and subdivision buffers can be found by directly calculating the Morton index and adding to it the number of tiles in all levels before search level:
+
+```
+(TILING_SCHEME ** (LEVEL + 1)) - 1) / (TILING_SCHEME - 1) + MORTON(X, Y, Z)
+```
+
+where `TILING_SCHEME` is 4 for quadtree and 8 for octree.
+
+For sparse trees, the tile lookups can be performed using the following algorithm:
+
+```javascript
+function getIndex(level, x, y, z) {
+    const mortonIndex = morton(x, y, z);
+    if (level === 1) {
+        return 1 + mortonIndex;
+    }
+    // Level 1 Search
+    const levelOneCellMorton = mortonIndex >> 2;
+    if (sBuffer[1 + levelOneCellMorton] === 1) {
+        // Level 2... Search
+        return traverse(level, mortonIndex, 2, 5);
+    }
+    return -1;
+}
+
+function traverse(targetLevel, morton, currentLevel, levelOffset) {
+    const parentTileMorton = morton >> (2 ^ (targetLevel - currentLevel + 1));
+    const currentTileOffset = morton % 4;
+    const currentTileBufferOffset = levelOffset + jBuffer[currentLevel - 1].indexOf(parentTileMorton) + currentTileOffset;
+
+    if (targetLevel === currentLevel) {
+        return currentTileBufferOffset;
+    }
+
+    if (sBuffer[currentTileBufferOffset] === 1) {
+        return traverse(targetLevel, morton, currentLevel + 1, levelOffset + (jBuffer[currentLevel].length * tilingScheme));
+    } else {
+        return -1;
+    }
+}
+```
+
+### Samples
+
+#### Complete Octree
+
+##### Root Tileset - tileset.json
+
+```json
+{
+    "asset": {
+        "version": "2.0.0-alpha.0"
+    },
+    "geometricValue": 10000,
+    "extensions": {
+        "3DTILES_implicit_tiling": {
+            "boundingVolume": {
+                "box": [
+                    0, 0, 0,
+                    5, 0, 0,
+                    0, 5, 0,
+                    0, 0, 5
+                ]
+            },
+            "refine": "REPLACE",
+            "tilingScheme": "octree",
+            "contentExtension": "glb",
+            "tilesetExtension": "json",
+            "content": {
+                "levelOffset": 4,
```

Comment:
It feels awkward that in order to specify a default value for all tiles `levelOffset` needs to exceed the maximum level of the tileset. Makes sense with the current design, but feels off.

Reply:
I agree that this is problematic. Perhaps, we can make it so that if `levelOffsetFill` is specified without `levelOffset`, then we can assume that the `levelOffsetFill` value applies to all tiles till `maximumLevel` (?)